tests/playwright/pages/elementor-panel-tabs/content.js

```diff
@@ -125,4 +130,23 @@ export default class Content {
 		await textEditor.getByText( 'Tab Content' ).click();
 		await textEditor.locator( EditorSelectors.tabs.body ).fill( text );
 	}
+
+	async parseSrc( src ) {
```

Comment:
What does this do?

Reply:
Added jsdoc. This one parses a[src] attribute and gets query params and its values. So we can test that our controls set the correct params.